File: circulo.java
public class Circulo {
    private Punto centro;
    private float radio;

    public Circulo(Punto centro, float radio) {
        this.centro = centro;
        this.radio = radio;
    }

    @Override
    public String toString() {
        return "Círculo con centro en " + centro + " y radio " + radio;
    }

    public void dibujaCirculo() {
        System.out.println("Dibujando círculo con centro en " + centro + " y radio " + radio);
    }
}

File: circulo.py
class Circulo:
    def __init__(self, centro, radio):
        self.centro = centro
        self.radio = radio

    def __str__(self):
        return f"Círculo centradi en {self.centro} y radio {self.radio}"

    def dibujaCirculo(self):
        print(f"dibujando el circulo con centro {self.centro} y radio {self.radio}")
